tests/unit/hook_packages/terraform/hooks/prepare/test_resource_linking.py

```diff
@@ -0,0 +1,67 @@
+from copy import deepcopy
+from unittest import TestCase
+
+from parameterized import parameterized
+
+from samcli.hook_packages.terraform.hooks.prepare.layer_linking import _clean_references_list
+
+
+class TestLayerLinking(TestCase):
```

Comment:
Open question: should this remain as `TestLayerLinking` or be changed to `TestResourceLinking`? I have it as `TestResourceLinking` in my other PR, but if we wanted link other resources, this can remain as is (and I can change it in my PR to match this)

Reply:
good catch, I'll change it

Reply:
done